backend/app/controllers/backoffice/admins_controller.rb

```diff
@@ -16,5 +19,36 @@ def index
         end
       end
     end
+
+    def new
+    end
+
+    def create
+      if @admin.update admin_params.merge(password: GeneratePassword.new(50).call, ui_language: current_admin.ui_language)
```

Comment:
`ui_language ` is mandatory for `Admin`, but formular from design does not contain this field soo I have taken its value from current admin. Not sure if it is correct or if I should add this field to formular.

Reply:
I think this should be added. What do you think @agnessa?

Reply:
agree